IntelliCommand/Services/CommandScopeService.cs

```diff
@@ -175,11 +175,13 @@ private void ParseFile(string filePath)
                     
                     var name = xNameAttribute.Value;
 
-                    if (!this.scopeDefinitions.ContainsKey(id))
-                    {
-                        this.scopeDefinitions.Add(id, name);
-                    }
+//in old version:

```

Comment:
Please fix the formatting of the comment to be consistent with the rest of this file. Also, I'd remove the parts that talk about the old version and instead just have it explain why we're replacing the entry that has this ID (if present).